src/content/getusermedia/record/js/main.js

```diff
@@ -99,37 +80,35 @@ function handleDataAvailable(event) {
   }
 }
 
-function handleStop(event) {
-  console.log('Recorder stopped: ', event);
-}
-
 function startRecording() {
   recordedBlobs = [];
   let options = {mimeType: 'video/webm;codecs=vp9'};
   if (!MediaRecorder.isTypeSupported(options.mimeType)) {
-    console.log(options.mimeType + ' is not Supported');
+    console.log(`${options.mimeType} is not Supported`);
```

Comment:
Same comment about showing something on the screen here.
In this case, it could be OK to show the MIME type you end up with on the screen.